easy_infra.yml

```diff
@@ -72,13 +67,20 @@ commands:
     version: 1.3.4
     version_argument: version
   terratag:
+    helper: ["terraform"]
     version: v0.1.48
   tfenv:
     allow_filter:
     - match: exec
       position: 0
     file_extensions: *id001
+    helper: ["terraform"]
     security: *id002
     validation: *id003
     version: v3.0.0
     version_argument: --version
+environments:
```

Comment:
This new section maps environments like `aws` and `azure` to a list of commands like `aws-cli` and `azure-cli`.  It leaves things open to more commands in the future being relevant to certain environments, as well as lets us be more clear in the code when we're talking about a concept (supporting an environment like aws) vs a specific tool (like `aws-cli`) vs a command that invokes a tool (like `aws`)